strapi/strapi
# Refactor helper-plugin table to be composable

### What does it do?

- Creates a new component to eventually replace the DynamicTable component in the helper-plugin

### Why is it needed?

- It reduces prop drilling
- Gives more control to the developer

### How to test it?
- Be sure to yarn setup or yarn build the helper-plugin
- Go to the content manager, everything should work as it did before


### Note
- Creating this as a draft for now since there is still outstanding work listed below. But I would really appreciate a review now.
- I believe there is a bug with the ActionBar alignment that is on main and not related to this PR (not sure yet if it is coming from the DS or not):

![Screenshot 2023-05-23 at 09 54 45](https://github.com/strapi/strapi/assets/26598053/f9bc1cdc-ff94-4442-9fff-e24778db12bd)


### TODO
- [x] Update the storybook
